src/fidesops/schemas/dataset.py

```diff
@@ -62,14 +62,18 @@ class FidesCollectionKey(ConstrainedStr):
     def validate(cls, value: str) -> str:
 
         values = value.split(".")
-        if (
-            len(values) == 2
-            and FidesKey.validate(values[0])
-            and FidesKey.validate(values[0])
-        ):
-            return value
-
-        raise FidesValidationError(
+        try:
+            if (
+                len(values) == 2
+                and FidesOpsKey.validate(values[0])
+                and FidesOpsKey.validate(values[0])
+            ):
+                return value
+        except ValueError:
+            raise ValueError(
+                "FidesCollection must be specified in the form 'FidesKey.FidesKey'"
+            )
+        raise ValueError(
```

Comment:
I'm unsure the exact use case for this validator, so unsure if using `ValueError` is the correct approach here

Reply:
@eastandwestwind this gets hit in the `test_dataset_yaml_format` - we could add a test that looks at the failing cases: where 1) the FidesCollectionKey is in the incorrect format, "hello", for example (because we expect a dataset name and collection name combined with a period).  and 2) if the individual pieces aren't fides keys "bad-dataset-name:bad-collection-name"